### ISSUE TITLE
Ecosystem integration

### ISSUE THREAD

**Issue by berceanu** [yt-project/unyt#452]
I was surprised to see that `pint` already has integrations for `pandas` as well as `xarray`. I expected `unyt`, due to its compatibility with `numpy`, to be easier to integrate into the scientific Python ecosystem. Is this on the roadmap?

**Comment by neutrinoceros** [comment]
There is no roadmap, and unyt isn't actively developed at the moment, but to answer your question there has been an effort to integrate unyt with dask. Related features are not released yet but will be part of the upcoming unyt 3.0

I think we would welcome any similar effort towards better integration with pandas or xarray, though nobody has yet taken on to do that yet.

**Comment by berceanu** [comment]
I see, thank you for your reply. Will 3.0 be a maintenance/bugfix release then, since you said it is not being actively developed anymore?

**Comment by neutrinoceros** [comment]
Not quite. There are a couple, rather big, unreleased features: dask integration and NEP 18 implementation. Both of these were implemented some time ago, but are not being worked on at the moment.

To be clear, we very much intend to keep maintaining unyt, and new developments are absolutely welcome, as far as I am concerned.
